lighthouse-core/test/runner-test.js

```diff
@@ -92,6 +92,17 @@ describe('Runner', () => {
       }
     });
 
+    it('-A throws if the URL changes', async () => {
+      const settings = {auditMode: artifactsPath, disableDeviceEmulation: true};
+      const opts = {url: 'https://example.com', config: generateConfig(settings), driverMock};
+      try {
+        await Runner.run(null, opts);
+        assert.fail('should have thrown');
```

Comment:
won't this be caught by the try/catch and so never fail?

Reply:
yeah but the message won't match, it's the same weird pattern observed elsewhere in the tests :) jest has some much niftier matchers FWIW ;)

Reply:
oh right. That's confusing but ok :)